definitions/types/gcp-security-iam.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "IAM",
+  "description": "IAM Roles And Conditions",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^.*$": {
+      "type": "object",
+      "required": [
+        "role"
```

Comment:
Only requiring role, so `condition` is optional. Thoughts? 🤷

Reply:
Good question. On the resource the condition block is optional, we could make it dynamic so that if a condition isn't present it doesn't add the block.